db/external_sst_file_ingestion_job.cc

```diff
@@ -599,22 +599,27 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
   file_to_ingest->largest_internal_key =
       InternalKey("", 0, ValueType::kTypeValue);
   bool bounds_set = false;
+  bool log_data = db_options_.allow_data_in_errors;
```

Comment:
Can you think about improving this naming `log_data`? It's hard to imaging the variable controls the failure messages.